metadata-ingestion/source_docs/kafka-connect.md

```diff
@@ -58,20 +58,20 @@ sink:
+| `provided_configs`             |          |                            | Provided Configurations                                                                                                                                                                                                                                                |
+| `construct_lineage_workunits`  |    | `True`                     | Whether to create the input and output Dataset entities                                                                                                                                                                                                                |
+| `connector_patterns.deny`      |          |                            | List of regex patterns for connectors to include in ingestion.                                                                                                                                                                                                         |
+| `connector_patterns.allow`     |          |                            | List of regex patterns for connectors to exclude from ingestion.                                                                                                                                                                                                       |
+| `connector_pattern.ignoreCase` |     | `True`                     | Whether to ignore case sensitivity during pattern matching.                                                                                                                                                                                                            |
+| `env`                          |          | `"PROD"`                   | Environment to use in namespace when constructing URNs.                                                                                                                                                                                                                |
+| `platform_instance_map`        |     |                            | Platform instance mapping to use when constructing URNs. e.g.`platform_instance_map: { "hive": "warehouse" }`                                                                                                                                                          | 
+| `prefix_server_to_table_name`  |          | `False`                    | Whether to prefix Database server name to the table name. When multiple instances are ingested for a single platform then the platform_instance name in the recipe could be matched to the database server name in Kafka connect and there by making the lineage work. |
```

Comment:
what would the generated urns look like if both platform_instance map is specified and the prefix_server_to_table_name config is set to true?

Reply:
When `prefix_server_to_table_name` config is set to `True` the urn would look like
`urn:li:dataset:(urn:li:dataPlatform:postgres,{database_server_name}.{db_name}.{schema_name}.{table_name}, PROD)`

For the lineage to work with the data ingested from say Postgres the [database.server.name](https://debezium.io/documentation/reference/stable/connectors/sqlserver.html#sqlserver-topic-names) should be same as `platform_instance` in database ingestion recipe.

When multiple instances for a platform is available `platform_instance_map` cannot be specified for that platform so only `prefix_server_to_table_name` is sufficient.